aspnetcore/migration/proper-to-2x/index.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Migrate from ASP.NET to ASP.NET Core 2.0
+title: Migrating from ASP.NET to ASP.NET Core Reference Guide
```

Comment:
- We try to avoid gerunds, so change "Migrating" back to "Migrate".
- Titles should use sentence casing, as a best practice. Change this title to "Migrate from ASP.NET to ASP.NET Core". The "Reference Guide" portion seems unnecessary.